### PR TITLE
Story

### PR DESCRIPTION
**Pull request by Parsley-Afroman**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+				</div>
+			</section>
+		</header>
+		<main id="about" class="about">
+			<h1 class="page-banner">About</h1>
+			<section class="abt-bx">
+				<section id="me"class="abt-cont">
+					<h2>Me</h2>
+					<div id="abt-lft" class="abt-wrap">
+						<div class="txt-wrap">
```
Comment (charliecog) [inline index.html:50]: since the child element is a p tag (block level by default) then you dont need this wrapping div

File: style.css [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+	height: inherit;
+	min-width: 200px;
+	width:40%;
+	max-width: 500px;
+	border: 2px solid #000;
+	object-fit: cover;
+}
+header section .wrap {
+	font-family: Helvetica;
+	position: relative;
```
Comment (charliecog) [inline style.css:116]: we try and stay away from position relative for things that can be achieved with margin - I would swap this out

File: style.css [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+	font-size: 2rem;
+}
+.abt-cont a {
+	font-size: 2rem;
+	border: 2px solid #fff;
+}
+.abt-wrap {
+	display: flex;
+	height: 100%;
+	width: 100%;
```
Comment (charliecog) [inline style.css:216]: not sure this is doing anything

File: style.css [hunk 4]
```diff
@@ -0,0 +1,347 @@
...
+	padding-top: 3rem;
```
Comment (charliecog) [inline style.css:217]: to do all your padding in one line you can do padding: 3rem 3rem 0 3rem;

**Review by charliecog — COMMENTED** [review]
looks good - almost there

general comments:
- i think the image of you in the about section might need to wrap to its own line a little sooner when making the screen smaller to preserve aspect ratio
- found a few places where the css isnt affecting the code - please have another look through and tick them on and off

File: index.html (hunk 1 above)
Comment (Parsley-Afroman) [reply index.html:50]: It amended it slightly but not in a way I can't do by styling the p

File: style.css [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+/*Nav Styling*/
+nav {
+	width: 100%;
+	height: 5rem;
+	background-color: #000080;
+	position: fixed;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	z-index: 99;
```
[the remark this answers is not in the document]
Comment (Parsley-Afroman) [reply style.css:17]: done

File: style.css [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+	margin: 0 auto;
+}
+p {
+	font-size: 2rem;
+	font-family: Helvetica;
+}
+/*Styling the Header*/
+header {
+	height: 100vh;
+	width: 100%;
```
[the remark this answers is not in the document]
Comment (Parsley-Afroman) [reply style.css:95]: done

File: style.css (hunk 2 above)
Comment (Parsley-Afroman) [reply style.css:116]: Removed and added margin instead

File: style.css [hunk 7]
```diff
@@ -0,0 +1,347 @@
...
+.abt-cont p {
+	font-size: 2rem;
+}
+.abt-cont a {
+	font-size: 2rem;
+	border: 2px solid #fff;
+}
+.abt-wrap {
+	display: flex;
+	height: 100%;
```
[the remark this answers is not in the document]
Comment (Parsley-Afroman) [reply style.css:215]: removed height

File: style.css [hunk 8]
```diff
@@ -0,0 +1,347 @@
...
+	width: 100%;
+	height: 50vh;
+	position: relative;
+	top: 20%;
+	display: flex;
+	justify-content: space-around;
+}
+header .logo-bx {
+	height: inherit;
+	min-width: 200px;
```
[the remark this answers is not in the document]
Comment (Parsley-Afroman) [reply style.css:109]: This has been done for it to appear better for the sprint review

File: style.css [hunk 9]
```diff
@@ -0,0 +1,352 @@
...
+}
+/*Styling the Header*/
+header {
+	height: 100vh;
+	width: 100%;
+	padding-top: 5rem;
+	background-color: #665500;
+}
+header section {
+	width: 100%;
```
[the remark this answers is not in the document]
Comment (Parsley-Afroman) [reply style.css:100]: done

File: style.css [hunk 10]
```diff
@@ -0,0 +1,352 @@
...
+}
+@media only screen and (max-height: 500px){
+	header section .wrap {
+		position: relative;
+		top: 10%;
+	}
+}
+.about {
+	min-height: 100vh;
+	width: 100%;
```
[the remark this answers is not in the document]
Comment (Parsley-Afroman) [reply style.css:167]: done

File: style.css (hunk 3 above)
Comment (Parsley-Afroman) [reply style.css:216]: removed

File: style.css (hunk 4 above)
Comment (Parsley-Afroman) [reply style.css:217]: top tip, thanks

File: style.css [hunk 11]
```diff
@@ -0,0 +1,347 @@
...
+	.txt-wrap {
+		width: 100%;
+	}
+}
+#log-wrap {
+	height: 300px;
+	width: 200px;
+}
+.logo {
+	height: inherit;
```
[the remark this answers is not in the document]
Comment (Parsley-Afroman) [reply style.css:262]: removed it

File: style.css [hunk 12]
```diff
@@ -0,0 +1,347 @@
...
+	width: inherit;
```
[the remark this answers is not in the document]
Comment (Parsley-Afroman) [reply style.css:263]: removed the whole class style

File: style.css [hunk 13]
```diff
@@ -0,0 +1,296 @@
...
+	}
+	
+	header section article {
+		height: 50%;
+		position: relative;
+		top: 30%;
+	}
+	header .logo-bx {
+		height: 100%;
+		position: relative;
```
Comment (charliecog) [inline style.css:132]: remove this to help fix overlap bug

File: style.css [hunk 14]
```diff
@@ -0,0 +1,296 @@
...
+		top: 30%;
```
Comment (charliecog) [inline style.css:133]: remove this to help fix overlap bug

**Review by charliecog — APPROVED** [review]

**Pull request merged**